src/canon/docs_test.jl

```diff
@@ -0,0 +1,26 @@
+
+db = [("Test One", "https://www.google.com", "testone", quote x="AAAA" end)
+     ,("Oscillator", "https://www.archlinux.org", "oscillator", quote y="BBBB" end)]
+
+macro create_pode(modelname,source,variable,pode) 
+  quote 
+    local v1 = $(esc(modelname))
+    local v2 = $(esc(source))
+    local v3 = $(string(pode))
+
+    """    $v1
+
+    [Source]($v2)
+
+    Model:
+
+    $v3
+
+    """
+  end
+end 
+
+for (modelname,link,variable,pode) ∈ db
+    @eval @create_pode $modelname $link $variable $pode
```

Comment:
If you replace `@create_pode` with a function `create_pode` then the line here can be 

`eval(create_pode(modelname, link, variable, pode))`